### PR TITLE
wd_core.au3 - cleanup

### PR DESCRIPTION
**Pull request by mlipok**

### CODE REVIEW

File: wd_core.au3 [hunk 1]
```diff
@@ -748,15 +748,15 @@ Func _WD_ElementAction($sSession, $sElement, $sCommand, $sOption = Default)
 	$sCommand = StringLower($sCommand)
 
 	Switch $sCommand
-		Case 'name', 'rect', 'text', 'selected', 'enabled', 'displayed', 'screenshot', 'shadow', 'comprole', 'complabel'
+		Case 'complabel', 'comprole', 'displayed', 'enabled', 'name', 'rect', 'selected', 'shadow', 'screenshot', 'text'
```
Comment (mlipok) [inline wd_core.au3:751]: just reordered

File: wd_core.au3 [hunk 2]
```diff
@@ -774,7 +774,7 @@ Func _WD_ElementAction($sSession, $sElement, $sCommand, $sOption = Default)
 			$iErr = @error
 
 		Case Else
-			Return SetError(__WD_Error($sFuncName, $_WD_ERROR_InvalidDataType, "(Name|Rect|Text|Selected|Enabled|Displayed|Active|Attribute|Property|CSS|Clear|Click|Value|Screenshot|Shadow|CompRole|CompLabel) $sCommand=>" & $sCommand), 0, "")
+			Return SetError(__WD_Error($sFuncName, $_WD_ERROR_InvalidDataType, "(Active|Attribute|CompRole|CompLabel|Clear|Click|CSS|Displayed|Enabled|Name|Property|Rect|Selected|Shadow|Screenshot|Text|Value) $sCommand=>" & $sCommand), 0, "")
```
Comment (mlipok) [inline wd_core.au3:777]: just reordered

File: wd_core.au3 [hunk 3]
```diff
@@ -748,15 +748,15 @@ Func _WD_ElementAction($sSession, $sElement, $sCommand, $sOption = Default)
...
+		Case 'complabel', 'comprole', 'displayed', 'enabled', 'name', 'rect', 'selected', 'shadow', 'screenshot', 'text'
 			$sResponse = __WD_Get($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/element/" & $sElement & "/" & $sCommand)
 			$iErr = @error
 
 		Case 'active'
 			$sResponse = __WD_Get($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/element/" & $sCommand)
 			$iErr = @error
 
-		Case 'attribute', 'property', 'css'
+		Case 'attribute', 'css', 'property'
```
Comment (mlipok) [inline wd_core.au3:759]: just reordered

File: wd_core.au3 [hunk 4]
```diff
@@ -1613,6 +1613,19 @@ Func __WD_EscapeString($sData)
 	Return SetError($_WD_ERROR_Success, 0, $sEscaped)
 EndFunc   ;==>__WD_EscapeString
 
+; #INTERNAL_USE_ONLY# ===========================================================================================================
+; Name ..........: __WD_TranslateQuotes
+; Description ...: Translate " into '
```
Comment (mlipok) [inline wd_core.au3:1618]: Please check this header

File: wd_core.au3 [hunk 5]
```diff
@@ -980,7 +980,7 @@ EndFunc   ;==>_WD_GetSource
 
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_Cookies
-; Description ...: Gets, sets, or deletes the session's cookies.
+; Description ...: Gets and sets, or deletes the session's cookies.
```
[the remark this answers is not in the document]
Comment (mlipok) [reply wd_core.au3:983]: Maybe like this:

`Gets, sets, or deletes the session's cookies.`
?

[the remark this answers is not in the document]
Comment (Danp2) [reply wd_core.au3:983]: How is that different from the "original" version?

[the remark this answers is not in the document]
Comment (mlipok) [reply wd_core.au3]: my mistake check "wording" commit

**Review by Danp2 — APPROVED** [review]

**Pull request merged**